econml/inference/_inference.py

```diff
@@ -462,9 +471,12 @@ def fname_transformer(x):
         else:
             def fname_transformer(x):
                 return x
+        # d_t=1 here since we measure the effect across all Ts
```

Comment:
I don't see why it should be a problem to pass a treatment name. This seems a logic that should be fixed inside the inference resutls if it's problematic to pass in a treamtnet name

Reply:
Fixed by setting `d_t=None` when summarizing over all treatments.